contentcuration/contentcuration/frontend/shared/views/channel/ChannelModal.vue

```diff
@@ -269,46 +269,81 @@
     // will never be rendered.
     beforeMount() {
       const channelId = this.$route.params.channelId;
+
+      // If there's no channelId (i.e., it's a new channel), skip verifications
+      if (!channelId) {
+        //this.isNew = true;
+        this.header = 'New Channel'; // Default header for a new channel
+        return; // Skip channel verification
+      }
+
+      // Existing channel, verify it
       return this.verifyChannel(channelId)
         .then(() => {
-          this.header = this.channel.name; // Get channel name when user enters modal
+          this.header = this.channel.name;
           this.updateTitleForPage();
           if (!this.isNew) {
             this.$refs.detailsform.validate();
           }
         })
-        .catch(() => {});
+        .catch(() => {
+          // Handle errors if needed
+        });
     },
     mounted() {
-      // Set expiry to 1ms
-      this.header = this.channel.name; // Get channel name when user enters modal
+      if (this.isNew) {
+        // For a new channel, set a default header
+        this.header = 'New Channel';
```

Comment:
This needs to use our UI strings. For example, using `this.$tr` with the preexisting string key in the `$trs` object of the component.